lib/quadquizaminos_web/live/contests_live.ex

```diff
@@ -111,11 +111,10 @@ defmodule QuadquizaminosWeb.ContestsLive do
       """
     else
       ~L"""
-      <button class= "<%= if contest.end_time, do: 'not_allowed' %>" phx-click="start" phx-value-contest='<%= contest.name %>' <%= if contest.end_time, do: 'disabled' %> >Start</button>
+      <button class= "<%= if contest.end_time, do: 'not_allowed' %>" phx-click="start" phx-value-contest='<%= contest.name %>' <%= if contest.end_time, do: 'disabled' %>><i class="fas fa-play-circle fa-2x"></i></button>
```

Comment:
```suggestion
      <a class= "<%= if contest.end_time, do: 'not_allowed' %>" phx-click="start" phx-value-contest='<%= contest.name %>' <%= if contest.end_time, do: 'disabled' %>><i class="fas fa-play-circle fa-2x"></i></a>
```
Replace the `<button>` with `<a>`  to avoid surrounding the icon with a button

![image](https://user-images.githubusercontent.com/43263401/126960418-f05f3b02-0051-465b-a103-81ef7e783648.png)

Reply:
As indicated on slack, you are okay with new styles of buttons